unstructured/ingest/v2/processes/embedder.py

```diff
@@ -58,6 +58,13 @@ def get_embedder(self) -> BaseEmbeddingEncoder:
             )
 
             return VertexAIEmbeddingEncoder(config=VertexAIEmbeddingConfig(**kwargs))
+        elif self.embedding_provider == "langchain-voyageai":
+            from unstructured.embed.voyageai import (
+                VoyageAIEmbeddingConfig,
+                VoyageAIEmbeddingEncoder,
+            )
+
+            return VoyageAIEmbeddingEncoder(config=VoyageAIEmbeddingConfig(**kwargs))
```

Comment:
this makes sense and follows convention. aside: there _has_ to be a better way to do this than an ever-growing if-else chain.. :/